src/ClashOfClans/Models/WarBase.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace ClashOfClans.Models
+{
+    public abstract class WarBase
+    {
+        public State? State { get; set; }
+
+        public int? TeamSize { get; set; }
+
+        public DateTime PreparationStartTime { get; set; }
+
+        public DateTime StartTime { get; set; }
+
+        public DateTime EndTime { get; set; }
+
```

Comment:
Can you please remove the empty line after the EndTime property (and before }-character) so that the class convention is in line with other classes.